net-nntp/sabnzbd/sabnzbd-1.1.1.ebuild

```diff
@@ -88,7 +88,8 @@ src_install() {
 	insopts -m 0755
 	doins SABnzbd.py
 
-	python_optimize "${D}usr/share/${PN}"
+	python_fix_shebang "${ED}usr/share/${PN}"
+	python_optimize "${ED}usr/share/${PN}"
```

Comment:
Please use `"${ED%/}/usr/share/${PN}"` instead as that will work with slashes possibly removed in some future EAPI.